skjemautfyller-eksempel/src/app.jsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { createStore, applyMiddleware, compose } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import rootReducer from './root/reducer';
+import Container from './root/container';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const enhancer = composeEnhancers(
+  applyMiddleware(thunk)
+);
+const store = createStore(rootReducer, enhancer);
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <Provider store={store}>
+        <Container />
+      </Provider>
+    );
+  }
+}
```

Comment:
Not a must at all, but React recommend using function components in new projects
```suggestion
const App = () => {
  return (
    <Provider store={store}>
      <Container />
    </Provider>
  );
}
```

Reply:
This is a part of the example code from skjemautfyller-eksempel, so don't think its too important to update it as it won't be a part of the formbuilder later @froydis98 . It was just merged in to make it easier to follow the example 🙂